app/src/app/[lng]/[inventory]/InventoryResultTab/TopEmissionsWidget.tsx

```diff
@@ -153,7 +153,7 @@ const TopEmissionsWidget = ({
                 </Heading>
               </Box>
               <EmissionsTable
-                topEmissions={results!.topEmissions.bySubSector}
+                topEmissions={results!?.topEmissions.bySubSector}
```

Comment:
Can you remove the ! operator since you're using ? as well?

Reply:
This leads to some type errors will handle in future ticket